<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vtn_principal</class>
 <widget class="QMainWindow" name="vtn_principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Estudiante</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="txt_nombre">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_email">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_semestre">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Seleccionar</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Primer Semestre</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Segundo Semestre</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tercer Semestre</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lbl_nombre">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_email">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Email:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_semestre">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Semestre:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_grabar">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>230</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Grabar</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_cedula">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cedula:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_cedula">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="sb_estado"/>
 </widget>
 <resources/>
 <connections/>
</ui>
